### PR TITLE
Mc questions and responses for JS

### PR DESCRIPTION
**Pull request by mytoalkoreath**

### CODE REVIEW

File: src/main/resources/static/js/createSurvey.js [hunk 1]
```diff
@@ -23,7 +24,37 @@ rangeInputForm =`
...
+     <textarea id="MCQuestionPrompt" name="questionPrompt" rows="2" cols="80"></textarea><br>
+	 <label for="MCChoicePrompt">Choice:</label><br>
+     <textarea id="MCChoice" name="MCChoice" rows="2" cols="80"></textarea><br>
+	 <button type="button" id="addChoice">Add Choice</button>
+	 <button type="button" id="deleteChoices">Delete All Choices</button>
+	 <button type="button" id="addMCQuestion">Add Question</button>
+	 
+    </div>`
 
+// Input HTML for setting the choices of a MC question
```
Comment (Samjenglish) [inline src/main/resources/static/js/createSurvey.js]: Can the commented out stuff be removed or is it supposed to be uncommentted?

Comment (mytoalkoreath) [reply src/main/resources/static/js/createSurvey.js]: Sorry, I just forgot to delete it

File: src/main/resources/static/js/createSurvey.js [hunk 2]
```diff
@@ -169,3 +217,22 @@ function addRangeQuestionToSurvey(questionPrompt, inputType, questionMin, questi
...
 }
+
+function addMCQuestionToSurvey(questionPrompt, inputType, options){
+ return new Promise((resolve, reject) => {
+    apiUrl = window.location.origin + '/api/question'
+    $.ajax({
+      url: apiUrl,
+      type: 'POST',
+      contentType: "application/json",
+      data: JSON.stringify({ survey: surveyHref, type: inputType, prompt: questionPrompt , options: options}),
```
Comment (Samjenglish) [inline src/main/resources/static/js/createSurvey.js]: options should be called choices to match the DB entity

Comment (mytoalkoreath) [reply src/main/resources/static/js/createSurvey.js]: Sure I'll make that change

File: src/main/resources/static/js/createSurvey.js [hunk 3]
```diff
@@ -56,6 +77,8 @@ $("#createSurveyForm").submit(function(e){
 				promises.push(addQuestionToSurvey(questions[i].prompt, questions[i].inputType))
 			}else if(questions[i].inputType == 'NUMBER'){
 				promises.push(addRangeQuestionToSurvey(questions[i].prompt, questions[i].inputType, questions[i].min, questions[i].max))
+			}else if(questions[i].inputType == 'MC'){
+				promises.push(addMCQuestionToSurvey(questions[i].prompt, questions[i].inputType, questions[i].options))
 			}
```
Comment (nikolaerak) [inline src/main/resources/static/js/createSurvey.js]: questions.choices instead of options

Comment (mytoalkoreath) [reply src/main/resources/static/js/createSurvey.js]: Fixed, thanks

**Pull request merged**